aptos-move/framework/cached-packages/src/aptos_token_sdk_builder.rs

```diff
@@ -475,8 +475,8 @@ pub fn token_mint_script(
 }
 
 /// mutate the token property and save the new property in TokenStore
-/// if the token property_version is 0, we will create a new property_version per token and store the properties
-/// if the token property_version is not 0, we will just update the propertyMap
+/// if the token property_version is 0, we will create a new property_version per token to generate a new token_id per token
```

Comment:
Where did these changes come from?

Reply:
I think it's rebuild for whatever reason, it didn't get updated for the pr introduced the change